packages/editor/src/editor-integration/create-basic-plugins.tsx

```diff
@@ -47,6 +49,12 @@ export function createBasicPlugins(props: Required<PluginsConfig>) {
       visibleInSuggestions: true,
       icon: <IconSpoiler />,
     },
+    {
+      type: EditorPluginType.DropzoneImage,
+      plugin: createDropzoneImagePlugin(),
+      visibleInSuggestions: true,
+      icon: <IconDropzones />,
+    },
```

Comment:
This would make the plugin immediately public in the integrations on the next update. Not sure this is intended.